src/spikeinterface/curation/remove_duplicated_spikes.py

```diff
@@ -82,8 +82,7 @@ def get_unit_spike_train(
         if end_frame == None:
             end_frame = spike_train[-1] if len(spike_train) > 0 else 0
 
-        start = np.searchsorted(spike_train, start_frame, side="left")
-        end = np.searchsorted(spike_train, end_frame, side="right")
+        start, end = np.searchsorted(spike_train, [start_frame, end_frame + 1], side="left")
```

Comment:
same here you change the side for end. please go back to the previous

Reply:
done